notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb

```diff
@@ -0,0 +1,1461 @@
+{
```

Comment:
<p>Vertex Model Artifact -&gt; Vertex AI Model artifact</p><p> </p><p>For links, use format</p><p> </p><p>Learn more about [title](link).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
updated.